src/status_im/ui/screens/about_app/views.cljs

```diff
@@ -8,7 +8,8 @@
             [re-frame.core :as re-frame]))
 
 (views/defview about-app []
-  (views/letsubs [version [:get-app-version]]
+  (views/letsubs [app-version [:get-app-short-version]
```

Comment:
there is no need to use subscription here because it's a constant

Reply:
Yeah, i just tried to keep it consistent :)